spec/default_spec.rb

```diff
@@ -9,6 +9,36 @@
       it 'converges successfully' do
         expect { chef_run }.to_not raise_error
       end
+      it do
+        expect(chef_run.node['varnish']['version']).to eq(3)
+      end
+      it do
+        expect(chef_run.node['varnish']['log_daemon']).to eq(false)
+      end
+      it do
+        expect(chef_run).to include_recipe('varnish')
+      end
+      it do
+        expect(chef_run).to create_template('/etc/sysconfig/varnishncsa').with(
+          source: 'varnishncsa.erb',
+          variables: { directives: chef_run.node['osl-varnish']['ncsa_directives'] }
+        )
+      end
+      it do
+        expect(chef_run).to render_file('/etc/sysconfig/varnishncsa')
```

Comment:
You need to also add ``.with_content()`` and and verify it's being formatted correctly from the template based on the attribute.